### DISCUSSION TITLE
How to run an individual workflow using graphrag

### DISCUSSION THREAD

**Discussion by vinayybhore** [microsoft/graphrag#922] [Algorithm + Paper]
I am trying to run the workflow until stage 2 where I just need to extract the entities and relationships from a text document,
I tried skipping workflows in the settings.yml but it didn't work and proceed with the next steps,
<img width="1106" alt="image" src="https://github.com/user-attachments/assets/46ee2868-5570-4f18-811a-29737c16e6be">
Can somebody please help?

**Comment by ultrageopro** [comment]
same question

**Comment by EmmittXu** [comment]
I tried --resume and skip_workflows, both not working. But here is a hardcode way out, comment "_get_workflow_list" function
![截屏2025-03-11 下午4 29 37](https://github.com/user-attachments/assets/af52543b-ee75-4e82-8804-ff9fb81d2cde)

**Reply by hejiang1972** [reply]
your graphrag version ? i could not find "_get_workflow_list" function

**Reply by hejiang1972** [reply]
oh , i found it. you are handsome!!!
